versioned_docs/version-3.2.x/event_triggers.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: event_triggers
+slug: /event_triggers
+title: Event Triggers
+description: Guide for setting up ports for equipment used in the clinic
+---
+
+## BrainVision Trigger Box setup
+
+Follow the TriggerBox setup instructions in the BrainVision Trigger Box manual. Plug the TriggerBox into the computer running the task. Check your operating system’s device list to identify the COM port that the TriggerBox is connected to. Create a new system environment variable:
+
+```shell
+COMNAME
+```
+
+and set the COM port to the correct value (e.g., COM3).
+
+## Open Source Event Trigger setup
+
+Details on how to make the open source event trigger and photodiode can be found [here](https://github.com/neuromotion/USB-event-marker).
+
+Connect the open source event trigger to the computer running the task using a USB to micro-USB cable. Check your operating system’s USB device list to identify the product ID of the teensyduino event marker.
+
+Create a new system environment variable: `EVENT_MARKER_PRODUCT_ID` and set to the product ID of the event marker.

```

Comment:
There's a bunch of `.env` files for some "defaults" that I think is intuitive enough. It's definitely something we have plans to rewrite though, so that will come